1-org/modules/centralized-logging/main.tf

```diff
@@ -185,3 +212,48 @@ resource "google_pubsub_topic_iam_member" "pubsub_sink_member" {
   role    = "roles/pubsub.publisher"
   member  = module.log_export["${each.value}_pub"].writer_identity
 }
+
+################################
+#-----------------------------------------------------------#
+# Log Bucket Service account IAM membership billing account #
+#-----------------------------------------------------------#
+
+resource "google_project_iam_member" "billing_account_member_logbucket" {
```

Comment:
Since this 3 `google_project_iam_member` resources are granting the same role, with same dependencies for different identities, how about make a for_each iterating by the logbucket, storage and pubsub `writer_identity`?